### PR TITLE
Modifications to Enableable

### PR DESCRIPTION
**Pull request by kingthorin**
Within org.zaproxy.zap.utils:
* Add interface EnableableInterface (Extracted from Enableable).
* Enableable now implements EnableableInterface.

Within org.zaproxy.zap.view:
* AbstractMultipleOptionsTableModel now leverages EnableableInterface.
* AbstractMultipleOptionsTablePanel now leverages EnableableInterface.

### CODE REVIEW

File: src/org/zaproxy/zap/utils/EnableableInterface.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.zaproxy.zap.utils;
```
Comment (thc202) [inline src/org/zaproxy/zap/utils/EnableableInterface.java]: Should be added the license header.

File: src/org/zaproxy/zap/utils/EnableableInterface.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package org.zaproxy.zap.utils;
+
+public interface EnableableInterface {
+
+	boolean isEnabled();
+
+	void setEnabled(boolean enabled);
+
+	int hashCode();
```
Comment (thc202) [inline src/org/zaproxy/zap/utils/EnableableInterface.java]: This method (and equals) could be removed from the interface, not really needed.

File: src/org/zaproxy/zap/utils/EnableableInterface.java (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/utils/EnableableInterface.java]: Done

**Comment by thc202** [comment]
Looks good to me, thank you!

**Pull request merged**